docs/nodevertical.md

```diff
@@ -150,8 +150,27 @@ Period of time (in seconds) that cluster loader will wait for pods to come up to
 Default: `600`  
 Pass/fail criteria. Value to determine if NodeVertical workload executed in duration expected.
 
-## Smoke test variables
+### NODEVERTICAL_HEAVY
+Default: `false`
+Trigger nodevertical heavy workload. Rather than deploy sleep pods throughout all labeled nodes, this workload deploys a two layer application composed by a PostgreSQL database pod and another pod wih simple application that is able to perform several operations and store a result in the database. [Heavy node vertical app](https://github.com/rsevilla87/perfApp)
+
+### NODEVERTICAL_HEAVY_PROBE_ENDPOINT
+Default: `/ready`
+Readiness probe endpoint for the application deployed by the heavy nodevertical. The default `/ready` endpoint inserts a record with the current timestamp in the ts table of the DDBB. i.e. `INSERT INTO ts VALUES ('2006-01-02T15:04:05Z07:00')`
 
+Results obtained of running 2 pods (client + database) with the default `/ready` endpoint.
+```
+# oc adm top pods -n nodevertical
```

Comment:
Maybe add a note for the users to use this info to check if they have enough resources on the worker nodes to host 250 similar pods before running the nodevert heavy? Thoughts?